.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -7,6 +7,9 @@ body:
     attributes:
       label: Description
       description: A brief description of the question or issue, also include what you tried and what didn't work
+      value: |
+        ### What is your problem
```

Comment:
```suggestion
        ### What is your problem
        
```

Reply:
Thanks for the suggestion (cant believe I missed that), I committed that locally and pushed it into the feat/add branch if you dont mind